pkg/subctl/benchmark/latency.go

```diff
@@ -118,4 +129,30 @@ func runLatencyTest(f *framework.Framework, testParams benchmarkTestParams) {
 
 	framework.By(fmt.Sprintf("Waiting for the client pod %q to exit, returning what client sent", nettestClientPod.Pod.Name))
 	nettestClientPod.AwaitFinishVerbose(Verbose)
+	latencyValues := strings.Split(nettestClientPod.TerminationMessage, "\n")[2]
+	values := strings.Split(latencyValues, ",")
+
+	var value []interface{}
+	value = append(value, latencyTypes{
+		minLatency:      values[0],
+		meanLatency:     values[1],
+		maxLatency:      values[2],
+		stddevLatency:   values[3],
+		transactionRate: values[4],
+	})
+	latencyPrinter.Print(value)
```

Comment:
Doing this instead
```
latencyHeaders := strings.Split(nettestClientPod.TerminationMessage, "\n")[1]
headers := strings.Split(latencyHeaders, ",")
latencyValues := strings.Split(nettestClientPod.TerminationMessage, "\n")[2]
values := strings.Split(latencyValues, ",")
for i, v := range headers {
fmt.Println(fmt.Sprintf("%s:\t%s", v, values[i]))
}
```

prints a vertical table and does not hard-code any information.